### PR TITLE
moved loop animation to its own editor section

### PR DESCRIPTION
**Pull request by SYBIOTE**
## Summary

created new editor node
removed loop animation from model editor node
added loopAnimation to editor.json

## References

closes #_insert number here_

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

**Comment by SYBIOTE** [comment]
do i connect the loop animation component to the loop animation node editor in 
packages/editor/src/functions/PrefabEditors.tsx

File: packages/editor/src/components/properties/ModelNodeEditor.tsx [hunk 1]
```diff
@@ -153,20 +154,13 @@ export const ModelNodeEditor: EditorComponentType = (props) => {
...
       </InputGroup>
-      <InputGroup name="Loop Animation" label={t('editor:properties.model.lbl-loopAnimation')}>
-        <SelectInput
-          key={props.entity}
-          options={animationOptions.value}
-          value={loopAnimationComponent?.activeClipIndex}
-          onChange={onChangePlayingAnimation}
-        />
-      </InputGroup>
       <InputGroup name="Is Avatar" label={t('editor:properties.model.lbl-isAvatar')}>
```
Comment (HexaField) [inline packages/editor/src/components/properties/ModelNodeEditor.tsx:157]: Is avatar belongs to loop animation component too

**Review by HexaField — CHANGES_REQUESTED** [review]
We need to register this new editor in PrefabEditors.tsx instead of reference it inside of another editor

File: packages/editor/src/components/properties/ModelNodeEditor.tsx (hunk 1 above)
Comment (SYBIOTE) [reply packages/editor/src/components/properties/ModelNodeEditor.tsx:157]: yup thats why i asked, but we still need to reference it in model node editor right?

Comment (SYBIOTE) [reply packages/editor/src/components/properties/ModelNodeEditor.tsx]: added to PrefabEditors.tsx, made it reactive and moved is avatar as well

**Review by HexaField — APPROVED** [review]

**Comment by speigg** [comment]
Something in this commit breaks the studio when selecting an entity with a ModelComponent

**Pull request merged**